select.go

```diff
@@ -163,6 +164,8 @@ func (b *SelectStmt) Build(d Dialect, buf Buffer) error {
 			buf.WriteString(strconv.FormatInt(b.OffsetCount, 10))
 		}
 	}
+
+	b.settings.Build(d, buf)
```

Comment:
Should this check if there are any settings before building it?

Reply:
We don't need to